### PR TITLE
Fix issues caused by PlusRemote moving from Python to C++

### PR DESCRIPTION
**Pull request by Sunderlandkyl**
See https://github.com/openigtlink/SlicerOpenIGTLink/pull/36

### CODE REVIEW

**Comment by Sunderlandkyl** [comment]
Can be integrated once https://github.com/openigtlink/SlicerOpenIGTLink/pull/36 is integrated.

**Comment by Sunderlandkyl** [comment]
This can be merged now that the prerequisite pull requests have been integrated.

**Comment by Sunderlandkyl** [comment]
@lassoan

File: Guidelet/GuideletLib/UltraSound.py [hunk 1]
```diff
@@ -168,6 +168,15 @@ def setupScene(self):
       displayNode.SetAndObserveColorNodeID(colorNode.GetID())
       self.liveUltrasoundNode_Reference.SetAndObserveDisplayNodeID(displayNode.GetID())
 
+    self.plusRemoteNode = slicer.mrmlScene.GetFirstNodeByName('PlusRemoteNode')
```
Comment (lassoan) [inline Guidelet/GuideletLib/UltraSound.py]: Should this be a singleton node?

Comment (Sunderlandkyl) [reply Guidelet/GuideletLib/UltraSound.py]: Done!

**Pull request merged**